bin/cylc-cat-log

```diff
@@ -321,14 +311,12 @@ def main():
             sys.exit(res)
         return
 
-    if options.by_remrun:
-        # If invoked remotely by ssh, start a watcher thread to kill me if the
-        # parent ssh connection dies.
-        t = threading.Thread(target=suicide)
-        t.start()
-
     suite_name = args[0]
-    mode = MODES[options.mode]
+    try:
+        mode = MODES[options.mode]
+    except KeyError:
+        # User chose log form.
+        mode = options.mode
```

Comment:
There are later usages of `options.mode` which means `cylc cat-log <suite> -m tail` results in traceback.